engine/src/main/scala/cromwell/engine/backend/package.scala

```diff
@@ -6,8 +6,6 @@ import scala.concurrent.{ExecutionContext, Future}
 import scala.collection.immutable.Seq
 
 package object backend {
-  class WorkflowContext(val root: String)
```

Comment:
I don't mind putting it somewhere else, creating a file to have a one line case class seemed overkill that's why I put it there.